### ISSUE TITLE
logits and labels must have the same first dimension, got logits shape [64,8] and labels shape [512]

### ISSUE THREAD

**Issue by herbiel** [KosminD/YAMNet_transfer#2]
i follow your step,but get this problem.can yo help  me ?
